examples/pom.xml

```diff
@@ -19,58 +19,58 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-
-    <parent>
-        <groupId>com.cloudera.livy</groupId>
-        <artifactId>livy-main</artifactId>
-        <version>0.3.0-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
-    </parent>
+  <modelVersion>4.0.0</modelVersion>
 
+  <parent>
     <groupId>com.cloudera.livy</groupId>
-    <artifactId>livy-examples</artifactId>
+    <artifactId>livy-main</artifactId>
     <version>0.3.0-SNAPSHOT</version>
-    <packaging>jar</packaging>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <groupId>com.cloudera.livy</groupId>
+  <artifactId>livy-examples</artifactId>
+  <version>0.3.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
 
-    <properties>
-        <skipTests>true</skipTests>
-        <skip>true</skip>
-        <skipDeploy>true</skipDeploy>
-    </properties>
+  <properties>
+    <skipTests>true</skipTests>
+    <skip>true</skip>
+    <skipDeploy>true</skipDeploy>
+  </properties>
 
-    <dependencies>
```

Comment:
It seems you're just changing indentation here... or are you changing something else too?

Please avoid noisy changes. I know the indentation is not consistent in all pom files but it would be better to fix that separately, otherwise it's hard to know what's changing.